### PR TITLE
Kj/pattern building

### PR DESCRIPTION
**Pull request by kelseythejackson**

### CODE REVIEW

File: docs/src/content/docs/experimental/patterns.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  openButton.addEventListener('click', () => drawer.show());
+  closeButton.addEventListener('click', () => drawer.hide());
+</script>
+```
+
+## Hero
+
+```html:preview
+<wa-button>HI</wa-button>
+</script>
```
Comment (KonnorRogers) [inline docs/src/content/docs/experimental/patterns.md]: Is this supposed to be here?

File: docs/src/content/docs/experimental/patterns.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+### Incentives
+
+```html:preview
+
+
+<style>
+
+</style>
+```
```
Comment (KonnorRogers) [inline docs/src/content/docs/experimental/patterns.md]: Am I missing something here?

File: docs/src/content/docs/experimental/patterns.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    <wa-switch class="wa-invalid">More awesome</wa-switch><br><br>
+    <wa-radio-group class="wa-invalid" label="Select an option" name="a" value="1">
+      <wa-radio value="1">Option 1</wa-radio>
+      <wa-radio value="2">Option 2</wa-radio>
+      <wa-radio value="3">Option 3</wa-radio>
+    </wa-radio-group><br>
+    <wa-button variant="brand">Submit Form</wa-button>
+  </div>
+</div>
+```
```
Comment (KonnorRogers) [inline docs/src/content/docs/experimental/patterns.md]: This looks a lot like the "Form Control Validation" section

https://webawesome-git-kj-pattern-building-font-awesome.vercel.app/experimental/form-validation/

Is the plan to move it from "form-validation" to this patterns section?

**Review by KonnorRogers — COMMENTED** [review]
This feels more like a placeholder PR, I'm assuming the intention is to fill out these pages?

File: docs/src/content/docs/experimental/patterns.md (hunk 1 above)
Comment (kelseythejackson) [reply docs/src/content/docs/experimental/patterns.md]: Oop, no it's not. I'll get rid of that.

File: docs/src/content/docs/experimental/patterns.md (hunk 2 above)
Comment (kelseythejackson) [reply docs/src/content/docs/experimental/patterns.md]: Hmm, no you're not. Gone

File: docs/src/content/docs/experimental/patterns.md (hunk 3 above)
Comment (kelseythejackson) [reply docs/src/content/docs/experimental/patterns.md]: I believe those were duplicates when I was making pages. Gone

**Pull request merged**